internal/cmd/ske/credentials/kubeconfig/login/login.go

```diff
@@ -0,0 +1,223 @@
+package login
+
+import (
+	"context"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"time"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/cache"
+	"k8s.io/client-go/rest"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/ske/client"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/ske"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	clientauthenticationv1 "k8s.io/client-go/pkg/apis/clientauthentication/v1"
+	"k8s.io/client-go/tools/auth/exec"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type inputModel struct {
+	ProjectId   string
+	ClusterName string
+	CacheKey    string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "login",
+		Short: "login plugin for kubectl",
+		Long:  "login plugin for kubectl to create a short-lived kubeconfig to authenticate against a STACKIT Kubernetes Engine (SKE) cluster. To get a kubeconfig to use with the login command use the 'kubeconfig create' command",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				"login to a SKE cluster specified in the kubeconfig",
+				"$ kubectl get pod"),
```

Comment:
The examples should include the usage of the CLI command. If there are other specific follow-up commands to other CLIs that are relevant for the example, such as kubectl, it's fine to have it but I would expect the CLI command to also be present

So this one at least seems incomplete

Reply:
Then I think removing the example would be better, because this command can only be used together with the "login kubeconfig" and kubectl.